tests/e2e/creation_test.go

```diff
@@ -325,11 +330,17 @@ func (tc *testContext) testApplicationCreation(component components.ComponentInt
 			log.Printf("waiting for application deployments to be in Ready state.")
 			return false, nil
 		}
+
+		// It's ok not to have deployements for unmanaged component
+		if component.GetManagementState() != operatorv1.Managed {
+			return true, nil
+		}
+
 		// when no deployment is found
 		// check Reconcile failed with missing dependent operator error
 		for _, Condition := range tc.testDsc.Status.Conditions {
 			if strings.Contains(Condition.Message, "Please install the operator before enabling "+component.GetComponentName()) {
-				return true, err
+				return true, nil
 			}
```

Comment:
do we still want to have this check? if we already have the test env set to install dependent operator

Reply:
I would like not to have it, but preserved the functionality.